src/dotnet/commands/dotnet-migrate/MigrateCommand.cs

```diff
@@ -224,6 +230,21 @@ private void AddProject(string slnPath, string csprojPath)
             addCommand.Execute();
         }
 
+        private void RemoveProject(string slnPath, string projPath)
```

Comment:
This and `AddProject` appear to be almost identical.  Can they be refactored to share the logic?